scripts/deploy-operator.sh

```diff
@@ -33,3 +33,13 @@ operator-sdk run bundle $OPERATOR_BUNDLE_IMAGE_FULL_NAME -n $TNF_EXAMPLE_CNF_NAM
 # If short name "csv" is used, the call will fail the first time 
 # With long name the first time it will work and subsequent time it will work with long or short names 
 oc get clusterserviceversions.operators.coreos.com -n $TNF_EXAMPLE_CNF_NAMESPACE
+
+CSV_MATCH=$(oc get csv -n $TNF_EXAMPLE_CNF_NAMESPACE -ogo-template='{{ range .items}}{{.metadata.name}}{{end}}' 2>/dev/null | grep "nginx-operator.v0.0.1")
```

Comment:
Maybe we could merge this call with the one in line 35? That one is only meant to enable the "csv" short name.

Reply:
fixed